build_miniforge.sh

```diff
@@ -37,5 +37,6 @@ for TEST_IMAGE_NAME in ${TEST_IMAGE_NAMES}; do
   echo "============= Test installer on ${TEST_IMAGE_NAME} ============="
   docker run --rm \
     -v "$(pwd):${CONSTRUCT_ROOT}" -e CONSTRUCT_ROOT \
-     --platform "linux/${DOCKER_ARCH}" "${DOCKER_ARCH/\//}/${TEST_IMAGE_NAME}" /construct/scripts/test.sh
+    -v /etc/ssl/certs:/etc/ssl/certs:ro \
```

Comment:
Side-note: I am not sure about the source of CA certificate `libmamba 1` uses, but [`libmamba 2` uses the system's CA certificates by default now](https://github.com/mamba-org/mamba/blob/9c856fd4a53ebb5774cf0de84a9583c4538cd567/libmamba/src/download/downloader.cpp#L27-L100), so the trust store storing them now needs to be mounted.

Are there other scripts or docker image usage in miniforge or conda-forge for which the trust store will need to be mounted?

Reply:
> Are there other scripts or docker image usage in miniforge or conda-forge for which the trust store will need to be mounted?

No, everything depends on `openssl` package from conda-forge which uses `ca-certificates` package in conda-forge. So, `libmamba=2` behaviour is wrong.

Reply:
OK, so this might explain issues like https://github.com/mamba-org/mamba/issues/3570.

Reply:
Once downloaded, micromamba should just work "offline" no internet or certificates required.

Reply:
https://github.com/mamba-org/mamba/pull/3765 aims at resolving it.

Reply:
Now resolved with 2.0.6.